pulsar-zookeeper-utils/src/test/java/org/apache/pulsar/zookeeper/ZkIsolatedBookieEnsemblePlacementPolicyTest.java

```diff
@@ -137,9 +167,12 @@ public void testBasic() throws Exception {
 
         Set<BookieId> bookieToExclude = new HashSet<>();
         bookieToExclude.add(new BookieSocketAddress(BOOKIE1).toBookieId());
-        ensemble = isolationPolicy.newEnsemble(2, 2, 2, Collections.emptyMap(), bookieToExclude).getResult();
-        assertTrue(ensemble.contains(new BookieSocketAddress(BOOKIE4).toBookieId()));
-        assertTrue(ensemble.contains(new BookieSocketAddress(BOOKIE2).toBookieId()));
+        try {
```

Comment:
why the behaviour of this test is changing ?
was the test wrong ?

Reply:
for the previous, the `BOOKIE4` was not included into any region, so it's good for including BOOKIE4  in ensemble.
But now,  we will add BOOKIE to blacklistedBookies, so it will throw error because only BOOKIE2 available.

Reply:
What about adding a new test case instead of modifying and existing one?

It looks like we are losing an existing test case

Reply:
But this test case will not pass.

Reply:
I added a new test case in `testNonRegionBookie `

Reply:
good